### PR TITLE
Some partial keys have been edited

### PR DESCRIPTION
**Pull request by Ali-Yazdani**
In order to regard better alignment between partial keys, Some partial keys have been edited.

### CODE REVIEW

File: hijri/07-rajab.yml [hunk 1]
```diff
@@ -132,7 +132,7 @@ events:
       - شیعه
   sources:
     - https://fa.wikipedia.org/wiki/موسی_کاظم#حبس_و_کشته‌شدن
-- partial_key: "muhammad_first_revelation"
+- partial_key: "ascension_of_prophet_mohammad"
```
Comment (hramezani) [inline hijri/07-rajab.yml]: This `muhammad_first_revelation` is the [english wiki page of this event](https://en.wikipedia.org/wiki/Muhammad%27s_first_revelation). it is better to keep it as is.

Comment (Ali-Yazdani) [reply hijri/07-rajab.yml]: I saw the Wiki link and I think both are the same but the new one has a direct reference to the event. anyways your comment is right if we should focus on the Wiki page.

File: hijri/03-rabi_al-awwal.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ events:
     - اسلامی
   sources:
     - https://fa.wikipedia.org/wiki/هجرت_محمد
-- partial_key: "death_of_hassan_askari"
+- partial_key: "martyrdom_of_hassan_askari"
```
Comment (hramezani) [inline hijri/03-rabi_al-awwal.yml]: Based on [https://en.wikipedia.org/wiki/Hasan_al-Askari](url) `martyrdom_hasan_al_askari`

**Comment by hramezani** [comment]
@Ali-Yazdani I think it would be good to rewrite names base on their wiki pages. what do you think?

**Comment by Ali-Yazdani** [comment]
Sounds good, since the Wiki pages haven't a predefined pattern, so we will face some issues in the future while if we wanna define the specified pattern for partial keys.

**Comment by Ali-Yazdani** [comment]
> @Ali-Yazdani I think it would be good to rewrite names base on their wiki pages. what do you think?

Sounds good, since the Wiki pages haven't a predefined pattern, so we will face some issues in the future while if we wanna define the specified pattern for partial keys.

**Comment by hramezani** [comment]
> > @Ali-Yazdani I think it would be good to rewrite names base on their wiki pages. what do you think?
> 
> Sounds good, since the Wiki pages haven't a predefined pattern, so we will face some issues in the future while if we wanna define the specified pattern for partial keys.

As far as I know and based on @fzerorubigd comment, we don't have patterns for them and they just use to to generate unique key of output.
I don't think we want to define patterns for partial_key.

**Comment by fzerorubigd** [comment]
I believe this is ok, after merging #35 I have the plan to freeze the partial keys. (before starting the gregorian calendar) so please review again and merge it if its ok for merging

**Pull request merged**